apps/dirac/Additional_Input_Data/readme.txt

```diff
@@ -0,0 +1,14 @@
+The input files intended for this folder are too large for a github repository.
+These files can however be downloaded from the University of Edinburgh's DataSync service.
+
+Login to a local machine that has an internet browser.
+Browse to "https://datasync.ed.ac.uk/index.php/s/ju6knXo5TVchspd", the password is "dirac3bm".
```

Comment:
Yes, this is meant to be Public.

Reply:
Awesome – just checking 😄
If they're public it may be nice to have them on [Zenodo](https://zenodo.org/) or some other cloud storage location that can be `wget`-ed in on demand. Only a suggestion – maybe something for the future @giordano ?

Reply:
Strongly agreed.  If it's ok to have this data publicly accessible, something which doesn't require authentication and can be `wget`-ed straight away would be much better.  We really want the benchmarks to be run reproducibly, not that people have to go around fetching the data manually.  Zenodo looks like a good option: they offer [50 GiB per dataset](https://help.zenodo.org/), I think we're within that limit, right?